pkg/minikube/kubeconfig/config.go

```diff
@@ -59,7 +60,7 @@ func GetConfigPath() string {
 }
 
 // Cache system admin entries to be used to run oc commands
-func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string) error {
+func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string, isVersion3_6 bool) error {
```

Comment:
I don't get why you have to pass _isVersion3_6_ into this method. IMO you can keep this method as is and just change the clusterName parameter clusterup.go. Then the need to check the version stays in one spot.